frontend/components/inbox_statistics/TicketsChart.vue

```diff
@@ -57,7 +57,7 @@
     async mounted() {
       const url = this.inboxId
           ? `/api/inboxes/${this.inboxId}/statistics/tickets/count`
```

Comment:
This requests returns a 404 error on localhost (docker) with user examplecoordinator

Reply:
Fixed.